App/Redux/IpfsNodeRedux.js

```diff
@@ -4,10 +4,7 @@ import Immutable from 'seamless-immutable'
 /* ------------- Types and Action Creators ------------- */
 
 const { Types, Creators } = createActions({
-  createNodeRequest: ['path'],
```

Comment:
Creating and starting the node are one in the same, so just combine them into `start`